### DISCUSSION TITLE
p/6451a5dcebcc413b9e668a3c9004d1a4/

### DISCUSSION THREAD

**Discussion by giscus** [rxliuli/blog#43] [Announcements]
# p/6451a5dcebcc413b9e668a3c9004d1a4/

周报 2024-02-04这周每天维系了每天 7 个小时的睡眠时间，这让白天的精力好多了，工作日下午一直没出去玩，但周六还是出去玩了一天。  出行周四周五一直沉浸在某种不舒服的情绪之中，于是周六出去玩了，从学校那里知道了吉田神社有举办活动，于是便去了那里，也在那儿短暂的见了一个从北京来的推友。 上午先去了吉田神社，那儿正在举办节分祭，但由于到晚上 11 点才开始篝火，吾辈并没有真正等到晚上。 Go

https://blog.rxliuli.com/p/6451a5dcebcc413b9e668a3c9004d1a4/

**Comment by aldlss** [comment]
搜索的时候搜索到了博主4年前写的文章，看了看没想到现在也还在更新，还是祝博主在日本生活顺利

**Reply by rxliuli** [reply]
嘛，希望能帮到你一点点🤏就好，这也是写博客的初衷。

**Comment by miaopass-future** [comment]
不回家过年么

**Reply by rxliuli** [reply]
不，这边春节不放假
